src/common/transformers/DEFAULT.js

```diff
@@ -0,0 +1,32 @@
+export const defval = (value, alternative) => {
+    if (!value) {
```

Comment:
If value is a string, OK.
But if it can be a number, one should compare with `undefined` (but not sure what is given when no value is defined).